chef/data_bags/crowbar/bc-template-ceph.json

```diff
@@ -6,7 +6,9 @@
       "master": false,
       "disk_mode": "first",
       "config": {
-        "fsid" : ""
+        "fsid" : "",
+        "osds_in_total": 0,
```

Comment:
It is calculated automatically but we should allow to setup also it manually at lease in raw mode.